Applications/Spire/Source/UiViewer/FlatButtonTestWidget.cpp

```diff
@@ -0,0 +1,84 @@
+#include "Spire/UiViewer/FlatButtonTestWidget.hpp"
+#include <QFont>
+#include "Spire/Spire/Dimensions.hpp"
+#include "Spire/Spire/Utility.hpp"
+
+using namespace Spire;
+
+namespace {
+  auto BUTTON_SIZE() {
+    static auto size = scale(100, 26);
+    return size;
+  }
+
+  auto make_labeled_button(const QString& label, QWidget* parent) {
+    auto button = new FlatButton(label, parent);
```

Comment:
Why is so much work being done in this function instead of in the constructor for the ```FlatButton```?

Reply:
So far we have a couple of different styles for FlatButton so the default style isn't really set. I could make the style in the spec the default that would be set in the constructor.

Reply:
For this task, only implement the button specified in the ui_kit/button.xd. If no class exists implementing that component then write a new one for it.

Reply:
Or have a make_flat_button(...) function like the I did for the CheckBox.

Reply:
Yes that works too.